torch/_dynamo/symbolic_convert.py

```diff
@@ -2358,7 +2394,8 @@ def get_trace_call_log_str():
         except Exception as e:
             log.debug("FAILED INLINING %s", code)
             raise
-        assert tracer.symbolic_result is not None
+        except NestedGraphBreak:
```

Comment:
Hmm, do we need to check that the parent has no backedge in its current function body?

Reply:
I think you need to propagate this data on the graph break itself, if I understand you correctly.